packages/block-library/src/table/transforms.js

```diff
@@ -10,6 +10,7 @@ const tableContentPasteSchema = {
 			th: {
 				allowEmpty: true,
 				children: getPhrasingContentSchema(),
+				attributes: [ 'scope' ],
```

Comment:
According to https://www.w3schools.com/tags/att_td_scope.asp it seems scope attribute can also be used on td elements. Should we also add `attributes: [ 'scope' ]`, to td? Currently when we paste a table with a scope on td elements the scope gets removed if we manually add a scope in the code editor it is persisted.

Reply:
Thanks for the review @jorgefilipecosta. It does say on that page `The <td> scope attribute is not supported in HTML5.` I think it might be better not to encourage future use of it.

Reply:
Perhaps I should make sure the attribute can't be output on `td`s in both the save and edit functions as well 🤔.

I'll add another commit, should be easy to do.